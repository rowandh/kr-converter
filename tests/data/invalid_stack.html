
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-2-91903832</p></td>
                                <td height="26" ><p align="center">&nbsp;2025-02-07 오후 3:30:05</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>zxcvgfh (9 트리플)</b></p></td>
<!--								<td ><p align="center">&nbsp;304934</p></td>   -->
								<td ><p align="center">&nbsp;304,934</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>tzghsgzw</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣A ♠A <font color='red'>♥</font>10 ♠K ♣3 ♠5 ♣9 [A 원 페어]  
<br>* 시작 : [StageNo:91903832] [Credit:378,341원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[tzghsgzw]
<br>* 앤티: -1,000원(377,341원)
<br>* 홀 카드딜: ♣A(39) ♠A(0) [A 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:A 원 페어(♣A ♠A)]
<br>* 베팅: 콜 -1,000원(376,341원) - 베팅순서: [0][2]
[6702ms]
<br>* 베팅: [올인] (376,341원) Credit(0원) - 베팅순서: [0][10]
[5139ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(♣A♠A) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
# 공베팅 반환 [247,949원]<br>* 종료: WinMoney[57,563원] Credit[305,512원]
<br>* 결과: 패배 [족보:A 원 페어] [카드:♣A ♠A ♠K <font color='red'>♥</font>10 ♣9]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-72,829</font></p></td>
											<td style="word-break:break-all"><p align="right">305,512</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>zxssdv</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>K <font color='red'>◆</font>5 [K 탑]  (다이) (Big Blind)
<br>* 시작 : [StageNo:91903832] [Credit:168,471원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:150]
<br>* NICKNAME:[zxssdv]
<br>* 앤티: -1,000원(167,471원)
<br>* 홀 카드딜: <font color='red'>◆</font>K(25) <font color='red'>◆</font>5(17) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(<font color='red'>◆</font>K <font color='red'>◆</font>5)]
<br>* 베팅: [블라인드:BIG] [금액:1,000원] [Creadit:166,471원]
<br>* 베팅: 다이 [0](166,471원) - 베팅순서: [0][9]
[234ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>K<font color='red'>◆</font>5) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[166,471원]
<br>* 결과: 패배 [족보:K 탑] [카드:<font color='red'>◆</font>K <font color='red'>◆</font>5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">166,471</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>hjytrtrtr</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>10 <font color='red'>◆</font>8 [10 탑]  (다이) 
<br>* 시작 : [StageNo:91903832] [Credit:121,403원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[hjytrtrtr]
<br>* 앤티: -1,000원(120,403원)
<br>* 홀 카드딜: <font color='red'>◆</font>10(22) <font color='red'>◆</font>8(20) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(<font color='red'>◆</font>10 <font color='red'>◆</font>8)]
<br>* 베팅: [풀] (15,000원) Credit(105,403원) - 베팅순서: [0][3]
[2204ms]
<br>* 베팅: 다이 [0](105,403원) - 베팅순서: [0][11]
[2304ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>10<font color='red'>◆</font>8) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[105,403원]
<br>* 결과: 패배 [족보:10 탑] [카드:<font color='red'>◆</font>10 <font color='red'>◆</font>8] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-16,000</font></p></td>
											<td style="word-break:break-all"><p align="right">105,403</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>hdhgxvb</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠10 ♠6 [10 탑]  (다이) 
<br>* 시작 : [StageNo:91903832] [Credit:114,589원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[hdhgxvb]
<br>* 앤티: -1,000원(113,589원)
<br>* 홀 카드딜: ♠10(9) ♠6(5) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(♠10 ♠6)]
<br>* 베팅: 다이 [0](113,589원) - 베팅순서: [0][5]
[208ms]
<br>* 커뮤니티 카드 딜: H(♠10♠6) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[113,589원]
<br>* 결과: 패배 [족보:10 탑] [카드:♠10 ♠6] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">113,589</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Kjgfddfhh</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>7 ♣4 [7 탑]  (다이) 
<br>* 시작 : [StageNo:91903832] [Credit:62,258원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[kjgfddfhh]
<br>* 앤티: -1,000원(61,258원)
<br>* 홀 카드딜: <font color='red'>◆</font>7(19) ♣4(42) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(<font color='red'>◆</font>7 ♣4)]
<br>* 베팅: 다이 [0](61,258원) - 베팅순서: [0][4]
[2716ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7♣4) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[61,258원]
<br>* 결과: 패배 [족보:7 탑] [카드:<font color='red'>◆</font>7 ♣4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">61,258</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>zxcvgfh</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠9 <font color='red'>◆</font>9 <font color='red'>♥</font>10 ♠K ♣3 ♠5 ♣9 [9 트리플]  (Entry Fee)
<br>* 시작 : [StageNo:91903832] [Credit:100,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[zxcvgfh]
<br>* 앤티: -1,000원(99,000원)
<br>* 홀 카드딜: ♠9(8) <font color='red'>◆</font>9(21) [9 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:9 원 페어(♠9 <font color='red'>◆</font>9)]
<br>* 신규 참가비 [금액:1,000원] [Credit:98,000원]
<br>* 베팅: [하프] (35,000원) Credit(63,000원) - 베팅순서: [0][7]
[2433ms]
<br>* 베팅: [올인] (63,000원) Credit(0원) - 베팅순서: [0][12]
[869ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(♠9<font color='red'>◆</font>9) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[304,934원] Credit[304,934원]
<br>* 결과: 승리 [족보:9 트리플] [카드:♣9 ♠9 <font color='red'>◆</font>9 ♠K <font color='red'>♥</font>10]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>204,934</font></p></td>
											<td style="word-break:break-all"><p align="right">304,934</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>alpzlbnda</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>A <font color='red'>♥</font>K <font color='red'>♥</font>10 ♠K ♣3 ♠5 ♣9 [K 원 페어]  (Small Blind)
<br>* 시작 : [StageNo:91903832] [Credit:130,392원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[alpzlbnda]
<br>* 앤티: -1,000원(129,392원)
<br>* 홀 카드딜: <font color='red'>◆</font>A(13) <font color='red'>♥</font>K(38) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(<font color='red'>◆</font>A <font color='red'>♥</font>K)]
<br>* 베팅: [블라인드:SMALL] [금액:1,000원] [Creadit:128,392원]
<br>* 베팅: [하프] (84,000원) Credit(44,392원) - 베팅순서: [0][8]
[2168ms]
<br>* 베팅: [올인] (44,392원) Credit(0원) - 베팅순서: [0][13]
[1180ms]
<br>* 쇼다운 상태 진입: [턴:0]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>A<font color='red'>♥</font>K) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[0원]
<br>* 결과: 패배 [족보:K 원 페어] [카드:♠K <font color='red'>♥</font>K <font color='red'>◆</font>A <font color='red'>♥</font>10 ♣9]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-130,392</font></p></td>
											<td style="word-break:break-all"><p align="right">0</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>fghdidtye</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠7 ♠3 [7 탑]  (다이) 
<br>* 시작 : [StageNo:91903832] [Credit:519,101원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[dfghsweq]
<br>* 앤티: -1,000원(518,101원)
<br>* 홀 카드딜: ♠7(6) ♠3(2) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(♠7 ♠3)]
<br>* 베팅: 다이 [0](518,101원) - 베팅순서: [0][1]
[1075ms]
<br>* 커뮤니티 카드 딜: H(♠7♠3) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[518,101원]
<br>* 결과: 패배 [족보:7 탑] [카드:♠7 ♠3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">518,101</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>artkjfgnioks</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠4 <font color='red'>♥</font>2 [4 탑]  (다이) 
<br>* 시작 : [StageNo:91903832] [Credit:365,550원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[artkjfgnioks]
<br>* 앤티: -1,000원(364,550원)
<br>* 홀 카드딜: ♠4(3) <font color='red'>♥</font>2(27) [4 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:4 탑(♠4 <font color='red'>♥</font>2)]
<br>* 베팅: 다이 [0](364,550원) - 베팅순서: [0][6]
[4823ms]
<br>* 커뮤니티 카드 딜: H(♠4<font color='red'>♥</font>2) C (<font color='red'>♥</font>10♠K♣3) (♠5) (♣9)
<br>* 종료: WinMoney[0원] Credit[364,550원]
<br>* 결과: 패배 [족보:4 탑] [카드:♠4 <font color='red'>♥</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">364,550</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-2-91903832">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


