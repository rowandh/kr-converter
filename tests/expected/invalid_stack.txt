PokerStars Hand #15291903832:  Hold'em No Limit ($1000/$1000) - 2025/02/07 15:30:05 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: fghdidtye ($519101 in chips)
Seat 2: tzghsgzw ($378341 in chips)
Seat 3: hjytrtrtr ($121403 in chips)
Seat 4: Kjgfddfhh ($62258 in chips)
Seat 5: hdhgxvb ($114589 in chips)
Seat 6: artkjfgnioks ($365550 in chips)
Seat 7: zxcvgfh ($100000 in chips)
Seat 8: alpzlbnda ($130392 in chips)
Seat 9: zxssdv ($168471 in chips)
fghdidtye: posts the ante $1000
tzghsgzw: posts the ante $1000
hjytrtrtr: posts the ante $1000
Kjgfddfhh: posts the ante $1000
hdhgxvb: posts the ante $1000
artkjfgnioks: posts the ante $1000
zxcvgfh: posts the ante $1000
alpzlbnda: posts the ante $1000
zxssdv: posts the ante $1000
alpzlbnda: posts small blind $1000
zxssdv: posts big blind $1000
zxcvgfh: posts big blind $1000
*** HOLE CARDS ***
fghdidtye: folds
tzghsgzw: calls $1000
hjytrtrtr: raises $14000 to $15000
Kjgfddfhh: folds
hdhgxvb: folds
artkjfgnioks: folds
zxcvgfh: raises $21000 to $36000
alpzlbnda: raises $49000 to $85000
zxssdv: folds
tzghsgzw: raises $292341 to $377341 and is all-in
hjytrtrtr: folds
zxcvgfh: calls $63000 and is all-in
alpzlbnda: raises $66392 to $129392 and is all-in
Uncalled bet ($247949) returned to tzghsgzw
*** FLOP *** [Th Ks 3c]
*** TURN *** [Th Ks 3c] [5s]
*** RIVER *** [Th Ks 3c] [5s] [9c]
tzghsgzw collected $57563 from side pot
zxcvgfh collected $304934 from main pot
*** SUMMARY ***
Total pot $382784 Main pot $304934. Side pot $57563. | Rake $20287
Board [Th Ks 3c 5s 9c]
Seat 1: fghdidtye mucked [7s 3s]
Seat 2: tzghsgzw showed [Ac As] and won ($57563)
Seat 3: hjytrtrtr mucked [Td 8d]
Seat 4: Kjgfddfhh mucked [7d 4c]
Seat 5: hdhgxvb mucked [Ts 6s]
Seat 6: artkjfgnioks mucked [4s 2h]
Seat 7: zxcvgfh showed [9s 9d] and won ($304934)
Seat 8: alpzlbnda showed [Ad Kh] and lost
Seat 9: zxssdv mucked [Kd 5d]